src/elm/Main.elm

```diff
@@ -859,6 +866,39 @@ goUp id (model, prevCmd) =
   model ! [prevCmd]
     |> activate targetId
 
+goToTopOfColumn : String -> (Model, Cmd Msg) -> (Model, Cmd Msg)
```

Comment:
This function works correctly for going to the top of the current **group**, but not the top of the current column. Just rename it to `goToTopOfGroup`.

(tips for getting to the top of the *column* elsewhere).